### PR TITLE
Instructions for adding a certificate

### PR DESCRIPTION
**Pull request by gerardsn**
Should this be in the README or another file?

Everything above `## Supported key types` is just adding a newline at the end of every sentence.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,123 @@
...
+    "jwkthumbprint": "xaVihytdhiID5wqCOsQbbb2_GuIb0r7m1GUgPIdBDjo",
+    "reason": "TODO: Give me one reason, give me just one reason why"
+}
+```
+
+Copy this into `config/certs.json` and set the reason for blocking the certificate.
+
+
+## Sign denylist
+The key to sign the denylist is stored in the `NutsDenylist` vault on 1password in the the `PrivateKey` item.
```
Comment (woutslakhorst) [inline README.md:71]: `the the`

File: README.md [hunk 2]
```diff
@@ -1,28 +1,123 @@
...
+DENYLIST_PRIVATEKEY_PEM="<private-key>" make denylist
+```
+
+
+## Publish denylist
+The previous step updated the signed denylist at `denylist-out/denylist.jws`. (DO NOT CHANGE THIS FILE LOCATION/NAME)
+Assuming the certificate to block was added to `certs/`, validate that the new denylist actually identifies the certificate.
+```shell
+# validate that the added certifcate is on the denylist
+make test
```
Comment (woutslakhorst) [inline README.md:99]: I would have called it `make sure`. Just kidding.

**Review by woutslakhorst — APPROVED** [review]
one minor typo

File: README.md [hunk 3]
```diff
@@ -1,28 +1,123 @@
...
+The [default cache for jsdelivr](https://www.jsdelivr.com/documentation#id-caching) is 12 hours for files on Github branches.
+TODO: contact jsdelivr to see if/how we can speed this up.
+
+The denylist file can be found at https://cdn.jsdelivr.net/gh/nuts-foundation/denylist@main/denylist/denylist.jws. 
+This points to the `main` branch on the Github repo.
+
+
+## Troubleshooting
+- `error parsing key: failed to parse PEM encoded key: failed to decode PEM data`:  
+  PEM data in an env variable can be a bit iffy. The header and footer should be separated from the key using new line character `\n`.
```
Comment (beardedfoo) [inline README.md]: This is not an actual newline character, but rather a '\' character followed by an 'n' character. This is replaced with newline characters after the environment variable is read.

File: README.md (hunk 2 above)
Comment (gerardsn) [reply README.md:99]: must resist... urge.. to change..

**Review by beardedfoo — APPROVED** [review]

**Pull request merged**